internal/cloud/realm/dependencies_test.go

```diff
@@ -28,8 +28,15 @@ func TestRealmDependencies(t *testing.T) {
 		assert.Nil(t, err)
 
 		uploadPath := filepath.Join(wd, "testdata/dependencies_upload.zip")
+
+		_, err = client.GetDependenciesStatus(groupID, app.ID)
+		assert.NotNilf(t, err, "dependency status must return error before import")
```

Comment:
can we assert the equality on the actual error itself?

Reply:
Yes although the error message essentially comes from the `baas` server so if this will have to be somewhat hard-coded and potentially break if it changes on the server side.

Reply:
you're right in pointing out a brittle test, but in this case I think it's fine to have.  for one, these tests have been this repo's main line of defense when it comes to server-side changes that unknowingly have an impact on the CLI.  obviously for this error message, its probably not critical to the functionality of the CLI, however its a good practice to get into.